devito/operator/operator.py

```diff
@@ -268,6 +260,9 @@ def _lower(cls, expressions, **kwargs):
         """
         # Create a symbol registry
         kwargs.setdefault('sregistry', SymbolRegistry())
+        # Add lang-base kwargs
+        kwargs.setdefault('lang', cls._Target.lang())
```

Comment:
it should be spelled out as `language` for homogeneity

also, I'm surprised there may be cases in which it's unset ? upon `rcompile`? that's kinda weird